pkg/localdata/constant.go

```diff
@@ -64,6 +64,12 @@ const (
 	// EnvTag is the tag of the running component
 	EnvTag = "TIUP_TAG"
 
+	// EnvNameSSHPassPrompt is the variable name by which user specific the password prompt for sshpass
+	EnvNameSSHPassPrompt = "TIUP_SSHPASS_PROMPT"
+
+	// EnvNameNativeSSHClient is the variable name by which user can specific use natiive ssh client or not
+	EnvNameNativeSSHClient = "TIUP_USE_NATIVE_SSH"
```

Comment:
```suggestion
	EnvNameNativeSSHClient = "TIUP_NATIVE_SSH"
```

Reply:
I think we it's time to add a new subcommand `tiup env` to show all environment variables that TiUP aware. I have submit an issue https://github.com/pingcap/tiup/issues/632.